packages/core/computedAsync/index.md

```diff
@@ -83,3 +83,5 @@ const userInfo = computedAsync(
 - Just like Vue's built-in `computed` function, `computedAsync` does dependency tracking and is automatically re-evaluated when dependencies change. Note however that only dependency referenced in the first call stack are considered for this. In other words: **Dependencies that are accessed asynchronously will not trigger re-evaluation of the async computed value.**
 
 - As opposed to Vue's built-in `computed` function, re-evaluation of the async computed value is triggered whenever dependencies are changing, regardless of whether its result is currently being tracked or not.
+
+- The default value of the `shallow` already changed to `true` in > 9.10.x.
```

Comment:
This should probably be version 10 as it is a breaking change that won't be released until then

Reply:
```suggestion
```